spec/models/comment_spec.rb

```diff
@@ -67,4 +67,33 @@
 
   end
 
+  describe '#report!' do
+    let(:comment) { FactoryGirl.create(:comment) }
+    let(:user) { FactoryGirl.create(:user) }
+
+    it 'sets attention_requested to true' do
+      comment.report!("Vexatious comment", "Comment is bad, please hide", user)
+      expect(comment.attention_requested).to eq(true)
+    end
+
+    it 'sends a message a message to admins' do
+      message = double(RequestMailer)
+      allow(RequestMailer).to receive(:requires_admin).and_return(message)
+      expect(message).to receive(:deliver)
+      comment.report!("Vexatious comment", "Comment is bad, please hide", user)
```

Comment:
Would it be better to pull out the last mail from ActionMailer::Deliveries and check content? See ["Don't mock what you don't own"](http://felixclack.com/post/51541743233/dont-mock-what-you-dont-own). Since we're not passing the request mailer in to the subject, we don't control it.